src/Auth/Storage/StorageInterface.php

```diff
@@ -23,24 +24,24 @@
     /**
      * Read user record.
      *
-     * @return \ArrayAccess|array|null
+     * @return mixed \ArrayAccess, array or null
```

Comment:
The mixed is wrong here, before was correct. Why should there suddenly be more types?